content/_guides/managing-flows.md

```diff
@@ -0,0 +1,147 @@
+
+| **URL Parameter**                    | **Required** | **Description**                                   |
+|----------------------------------|--------------|---------------------------------------------------|
+| `FLOW_ID`                             | yes          | Flow ID |
+
+4\. To edit a Flow to some extent via the API, we will use the following request:
+
+`PATCH {{apiBaseUri}}/v2/flows/{FLOW_ID}`
+
+Below are request parameters:
+
+| **URL Parameter**                    | **Required** | **Description**                                   |
+|----------------------------------|--------------|---------------------------------------------------|
+| `FLOW_ID`                             | yes          | Flow ID |
+
+
+| **Payload Parameter**                    | **Required** | **Description**                                   |
+|----------------------------------|--------------|---------------------------------------------------|
+| `type`                             | yes          | Allowed value: `flow` |
+| `id`                             | yes          | Flow ID |
+| `attributes.name`                             | no         | Flow name |
+| `attributes.type`                             | no          | Flow type. Allowed values: `ordinary`, `long_running `|
+| `attributes.graph`                             | no          |  	Flow graph representing component connections |
+| `attributes.cron`                             | no          | CRON expression representing Flow timing |
+
+
+## Copy Flow, Switch Flow Type
+
+To copy a Flow within the same Workspace, navigate to see your Flows. Then click on the corresponding icon to open the settings menu of the selected Flow:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_7.png)  
+
+Alternatively, you can navigate to the Flow itself by clicking on its name:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_3.png)
+
+Then switch to *Implement* tab **(1)** and click the command **(2)** to copy it to clipboard, and [create a copy of the Flow via the API](/managing-workspaces):
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_8.png)  
+
+As a result, you get a copy of your Flow:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_9.png)  
+
+To switch between real-time and ordinary types, navigate to see your Flows. Then click on the corresponding icon to open the settings menu of the selected Flow:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_10.png)
+
+Switching back works the same way. Alternatively, you can navigate to the Flow itself by clicking on its name, switch to *Settings* tab **(1)**, scroll down and toggle the corresponding switch **(2)**:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_11.png)
+
+Note, that you can only switch Flow type if there's at least one published Flow version, and it is not running.
+
+## Additional Actions
```

Comment:
Again I would split this one into 2 subsections because we need to explain the `CRON` part in more details.